internal/anchor/reconciler.go

```diff
@@ -67,22 +67,23 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	// Get instance from apiserver. If the instance doesn't exist, do nothing and early exist because
 	// HCR watches anchor and (if applicable) has already updated the parent HC when this anchor was
 	// purged.
-	inst, err := r.getInstance(ctx, pnm, nm)
+	baseInst, err := r.getInstance(ctx, pnm, nm)
```

Comment:
Nit: I prefer `origInst` to focus on what's changed - I find "base" to be ambiguous here. To my eyes, "orig" makes it more clear that we'll be doing a diff on it later.